modules/ding_mkws/ding_mkws.module

```diff
@@ -0,0 +1,357 @@
+<?php
+/**
+ * @file
+ * Represents main hooks implementations and other common functions.
+ */
+
+include_once 'ding_mkws.fields.inc';
+
+/**
+ * Implements hook_libraries_info().
+ */
+function ding_mkws_libraries_info() {
+  $libraries = array();
+
+  $libraries['pz2'] = array(
+    'name' => 'Pazpar2 library',
+    'vendor url' => 'http://www.indexdata.com/pazpar2/',
+    'download url' => 'http://ftp.indexdata.dk/pub/pazpar2/pazpar2-1.12.5.tar.gz',
+    'version callback' => 'ding_mkws_version_callback',
+    'files' => array(
+      'js' => array('js/pz2.js'),
+    ),
+  );
+
+  $libraries['jsrender'] = array(
+    'name' => 'JS Render',
+    'vendor url' => 'https://github.com//BorisMoore/jsrender',
+    'download url' => 'https://github.com//BorisMoore/jsrender/archive/master.zip',
+    'version callback' => 'ding_mkws_version_callback',
+    'files' => array(
+      'js' => array('jsrender.min.js'),
+    ),
+  );
+
+  return $libraries;
+}
+
+/**
+ * Support function for detecting versions of library.
+ *
+ * @return bool
+ *   Status of version detection.
+ */
+function ding_mkws_version_callback() {
+  return TRUE;
+}
+
+/**
+ * Implements hook_menu().
+ */
+function ding_mkws_menu() {
+  $items = array();
+
+  $items['admin/config/ding/mkws'] = array(
+    'title' => 'MKWS settings',
+    'description' => 'Settings and defaults for MKWS implementation',
```

Comment:
I'd suggest you use a more descriptive title and description. Googling 'mkws' returns 'Milton Keynes Wargames Society' way ahead of 'MasterKey Widget Set'. And even THAT don't tell much about what it's actually used for...